drivers/sensor/tcn75a/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright 2022 Daniel DeGrasse <daniel@degrasse.com>
```

Comment:
Actually it's 2023 :)

Reply:
Code was written in 2022. I've updated copyrights.